frontend/coprs_frontend/coprs/templates/banner-include.html

```diff
@@ -0,0 +1,14 @@
+<div class="alert alert-danger" role="alert">
+    {% if outage_time and ticket %}
+        <b>Warning!</b> Scheduled outage of Copr servers - {{ outage_time }}. For more information
```

Comment:
hmm, if ticket is not specified then the main headline will not be displayed? Can you split it to two independent if?